### PR TITLE
dbeaver/pro#2338 Enhance unassociated entity location for ortho layout

### PR DESCRIPTION
**Pull request by serjiokov**
image

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

**Comment by E1izabeth** [comment]
Does it fix this issue [#2021](https://github.com/dbeaver/pro/issues/2021)?

File: plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/layout/algorithm/direct/OrthoDirectedGraphLayout.java [hunk 1]
```diff
@@ -136,10 +136,30 @@ private void drawMissedNodes(List<Node> islands, List<Node> missedNodes, TreeMap
...
+            if (width < node.width) {
+                width = node.width;
+            }
+            if (height < node.height) {
+                height = node.height;
+            }
+            node.x = offsetX;
+            node.y = offsetY;
+            virtColumns++;
+            if (virtColumns % 5 == 0) {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/layout/algorithm/direct/OrthoDirectedGraphLayout.java]: No magic constants, please

Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/layout/algorithm/direct/OrthoDirectedGraphLayout.java]: Done

**Review by ShadelessFox — APPROVED** [review]

**Comment by DariaMarkaryan** [comment]
When all parameters of tables are shown (View styles -> Check all parameters) some tables overlap (for orthogonal and Mikami routings)

https://github.com/dbeaver/dbeaver/assets/43252954/fcb0db73-b19a-4779-a286-2994c42e12cf

**Comment by DariaMarkaryan** [comment]
Verified

**Pull request merged**